File: diary-server/src/test/java/com/diary/server/DiaryServerApplicationTests.java
package com.diary.server;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class DiaryServerApplicationTests {

    @Test
    void mainClassPackageIsCorrect() {
        assertEquals("com.diary.server", DiaryServerApplication.class.getPackageName());
    }
}

File: diary-server/src/test/kotlin/com/diary/server/DiaryServerApplicationTests.kt
package com.diary.server

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class DiaryServerApplicationTests {

    @Test
    fun mainClassPackageIsCorrect() {
        assertEquals("com.diary.server", DiaryServerApplication::class.java.packageName)
    }
}
